DragaliaAPI/Models/Data/ManaNodes.cs

```diff
@@ -48,7 +48,7 @@ public static ManaNodes SetManaCircleNodesFromSet(IEnumerable<int> enumerable)
 
     public static SortedSet<int> GetSetFromManaNodes(ManaNodes flag)
     {
-        SortedSet<int> nodes = new();
+        SortedSet<int> nodes = new SortedSet<int>();
```

Comment:
what's wrong with the implicit `new()`?

Reply:
actually this may have been a change I did which was then not reflected in your branch, so it looks like you reversed it